Intel_SighPP_CMake/Intel_SighPP_CMake/src/interface_controller/api_controller.cpp

```diff
@@ -10,19 +10,17 @@ void ApiController::add_user(ApiUserInterface& user)
 	api_users_.push_back(&user);
 }
 
-void ApiController::new_items(const std::vector<PrioritisedClassificationResult>& results)
+void ApiController::new_items(const std::vector<ClassificationItem>& results)
 {
 	SPDLOG_INFO("Sending items to API users");
 
-	int time = std::clock();
+	auto time = std::clock();
 
 	SPDLOG_INFO("Testing Time counter, time is {} outside loop", std::to_string(time - duration));
-	if ((time - duration) > 2000)
-	{
```

Comment:
I want to know what is the initial purpose for (time - duration) > 2000)

Reply:
So this was before the colldown system was implemented and it basically stoped the service from sending data every frame. sent it two seconds instead